packages/web/components/earn/rewards/index.tsx

```diff
@@ -1,16 +1,14 @@
-import { Icon } from "~/components/assets";
 import { Button } from "~/components/buttons";
-import { Tooltip } from "~/components/tooltip";
 import { useTranslation } from "~/hooks";
 
 export const EarnRewards = () => {
   const { t } = useTranslation();
   return (
-    <div className="flex flex-col justify-between rounded-3x4pxlinset bg-osmoverse-850 px-8 pt-7 pb-9">
+    <div className="flex flex-col justify-between rounded-3x4pxlinset bg-osmoverse-850 px-6 pt-7 pb-6">
       <h5 className="text-lg font-semibold text-osmoverse-100">
         {t("earnPage.rewards")}
       </h5>
-      <div className="mt-5 flex flex-col gap-1">
+      {/* <div className="mt-5 flex flex-col gap-1">
```

Comment:
?

Reply:
same as before, i commented out the previous layout. Would you prefer if i delete the comments entirely?